KsApi/MockService.swift

```diff
@@ -116,8 +116,7 @@
     fileprivate let fetchUserProjectsBackedResponse: [Project]?
     fileprivate let fetchUserProjectsBackedError: ErrorEnvelope?
 
-    fileprivate let fetchUserResponse: User?
-    fileprivate let fetchUserError: ErrorEnvelope?
+    fileprivate let fetchUserResult: Result<User, ErrorEnvelope>?
```

Comment:
Updated this after some unit tests broke when I did a similar refactor for the `signInWithAppleResult` type.